plugins/kubernetes/app/models/kubernetes/deploy_executor.rb

```diff
@@ -321,12 +317,12 @@ def deploy(release_docs)
       end
     end
 
-    def deploy_to_cluster(release, deploys)
-      deploy(deploys)
-      successful = wait_for_resources_to_complete(release, deploys)
+    def deploy_to_cluster(release, release_docs)
+      deploy(release_docs)
+      successful = wait_for_resources_to_complete(release, release_docs)
       unless successful
-        show_failure_cause(release)
-        rollback(deploys)
+        show_failure_cause(release, release_docs)
```

Comment:
previously this showed all failed deploys ... which was never a real issue since if something failed the previous deploy would have failed ... , but this should be more correct